docs/application/native/guides/ui/dali/imageview-n.md

```diff
@@ -3,27 +3,80 @@
 - Tizen 2.4 and Higher for Mobile
 - Tizen 3.0 and Higher for Wearable
 
-The `Dali::Toolkit::ImageView` component displays an image.
+The `Dali::Toolkit::ImageView` is a control which displays an image. It's a common use is for showing an ImageVisual.
 
-**Figure: ImageView**
+In this tutorial the following subjects are covered:
+
+[Basic creation and usage](#1)<br>
+[Creation and usage with Visuals](#2)<br>
+[ImageView Properties](#3)<br>
+
+<a name="1"></a>
+## Basic creation and usage
+
+A basic instance of an ImageView is created with a file path:
+
+If the filename is to a Nine Path, GIF, SVG or regular image, then the respective Visual will be created and used.
+
+```
+ImageView imageView = ImageView::New("./images/image_view.svg");
+```
 
 ![ImageView](./media/image_view.png)
 
-## Creating an ImageView
+<a name="2"></a>
+## Creation and usage with Visuals
+
+To have full control over the Visual and what ImageView displays a Visual, Property::Map should be used.
 
-The image view is constructed by passing a `Dali::Image` object. The `Dali::Image` is an abstract base class with multiple derived classes, and the `Dali::ResourceImage` class is used for [loading an image from a file or URL](resources-n.md). The following example shows how to create an `ImageView` object:
+Below example uses the image and forces ATLASING off.
+Each Visual has it's own Properties which can be set.
 
 ```
-ImageView imageView = ImageView::New( myImageURL );
-// Or
-Image image = ResourceImage::New( myImageURL );
-ImageView imageView = ImageView::New( image );
+auto imageView = ImageView:New();
+
+imageView.SetProperty( ImageView::Property::IMAGE,
+                       Property::Map().Add( ImageVisual::Property::URL, "./images/apple.svg" )
+                                      .Add( ImageVisual::Property::ATLASING, false  ) );
 ```
 
-## Changing the Image
+To subsequently change a visual, then set the IMAGE property a new visual property map.
 
-The image object can be changed later by calling the `ImageView::SetImage()` function:
+```
+imageView.SetProperty( ImageView::Property::IMAGE,
+                       Property::Map().Add( ImageVisual::Property::URL, "./images/house.png" )
+                                      .Add( ImageVisual::Property::ATLASING, false  ) );
 
+imageView.SetParentOrigin( ParentOrigin::CENTER );
+imageView.SetResizePolicy( ResizePolicy::USE_NATURAL_SIZE, Dimension::ALL_DIMENSIONS );
 ```
-imageView.SetImage( newImage );
-```
\ No newline at end of file
+
+This image is displayed via an ImageView:
+
+<img src="./media/house.png" width="400" height="600">
+
+<a name="3"></a>
+## ImageView Properties
+
+ImageView has the following properties:
+
+| Property  | Type | Description |
+| ------------ | ------------ | ------------ |
+| IMAGE  | Map | map of properties associated with a given image. |
+| PRE_MULTIPLIED_ALPHA | bool | opacity adjusted image. |
+| PIXEL_AREA | Vector4 | sub area of image (Animatable). |
+
+> **Note**  
```

Comment:
It looks like the below contents are all in Note style. (from line 70 PreMultipliedAlpha to line 82). Is it correct?